module4/homework/appendNewRecipe/AppendNewRecipe.cpp

```diff
@@ -0,0 +1,74 @@
+#include "AppendNewRecipe.hpp"
+
+#include <iostream>
+#include <fstream>
+
+bool AppendNewRecipe(vecOfStrings steps, const listOfStrings& ingredients, const dequeOfPairs& amount){
```

Comment:
Don't copy vectOfStrings and also why you use aliases? This is unnecessary here.